examples/camera_streamer/camera_streamer.sh

```
@@ -133,6 +190,67 @@ ${_BOLD}EXAMPLES${_RESET}
+    local REMOTE_DIR="$REMOTE_BUILD_DIR/camera_streamer"
+    local HOST_CONFIG="$SCRIPT_DIR/$CONFIG"
+
+    if [[ ! -f "$HOST_CONFIG" ]]; then
+        log_error "Config not found: $HOST_CONFIG"
+        exit 1
+    fi
+
+    log_step "Pushing config to $REMOTE"
+    log_info "Config: $CONFIG"
+
+    ssh "$REMOTE" "mkdir -p $REMOTE_DIR/$(dirname "$CONFIG")"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)